### PR TITLE
Establish the http4s Code of Conduct.

### PR DESCRIPTION
**Pull request by rossabaker**
This proposal is not in response to any incident.  Our community has a track record of being kind and civil, and I'm proud of that.  However, the time has come:

1. We are growing, and we should be prepared for the growing pains.

2. We don't know who might have been reluctant to contribute in the past for lack of an explicit policy.

### CODE REVIEW

**Comment by rossabaker** [comment]
I invite, but don't require, members of @http4s/owners, @http4s/contributors, and @http4s/committers to add their names to conduct.md as community staff.

File: docs/src/jekyll/conduct.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Community staff will be happy to help participants contact local law
+enforcement or otherwise assist those experiencing harassment to feel safe. We
+value your participation.
+
+We expect participants to follow these rules in all http4s-supported venues.
+These venues include, but are not necessarily limited to: 
+
+- All GitHub repositories and comments under the [http4s organization](https://github.com/http4s)
+- The [http4s gitter channel](https://gitter.com/http4s/http4s)
+- The #typelevel IRC channel on Freenode,
```
Comment (bryce-anderson) [inline docs/src/jekyll/conduct.md]: I suspect this line is not an accident, correct?

**Comment by bryce-anderson** [comment]
:+1: LGTM.

**Comment by rossabaker** [comment]
I'll leave this open for a short while for co-signers or dissenters, but it will probably take a lot to dissuade me from this being a good idea.

File: docs/src/jekyll/conduct.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+These venues include, but are not necessarily limited to: 
+
+- All GitHub repositories and comments under the [http4s organization](https://github.com/http4s)
+- The [http4s gitter channel](https://gitter.com/http4s/http4s)
+
+## Community Staff
+
+If you have any questions or concerns, please contact a member of the community staff, such as:
+
+- [Bryce Anderson](bryce.anderson22@gmail.com)
```
Comment (stephenjudkins) [inline docs/src/jekyll/conduct.md]: Both or neither of these addresses should have the `mailto` prefix, probably

Comment (rossabaker) [inline docs/src/jekyll/conduct.md]: Yep. Thanks!

**Pull request merged**